<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check</string>
  </property>
  <widget class="QWidget" name="container">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="value">
               <number>24</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QListView" name="listView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_vod">
       <attribute name="title">
        <string>Video</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="mainBox">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="topBox">
            <item>
             <widget class="QScrollArea" name="view">
              <property name="minimumSize">
               <size>
                <width>670</width>
                <height>670</height>
               </size>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="viewForm">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>653</width>
                 <height>654</height>
                </rect>
               </property>
               <widget class="QLabel" name="viewData">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>800</width>
                  <height>600</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="midBox">
            <item>
             <widget class="QPushButton" name="btn_openFile">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Open File</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spa_3_cam">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSlider" name="slider_videoframe">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spa_4_cam">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lbl_frame">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>frame</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bottumBox">
            <item>
             <widget class="QComboBox" name="viewCbo_roi">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <property name="currentText">
               <string>0.5</string>
              </property>
              <item>
               <property name="text">
                <string>0.5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.75</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.5</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="spa_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_capture">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Capture</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_stop">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_pause">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Pause</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_play">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Play</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_cam">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="mainBox_cam">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="topBox_cam">
            <item>
             <widget class="QScrollArea" name="view_cam">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="viewForm_cam">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>668</width>
                 <height>669</height>
                </rect>
               </property>
               <widget class="QLabel" name="viewData_cam">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>47</width>
                  <height>12</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="midBox_cam">
            <item>
             <widget class="QPushButton" name="btn_open_cam">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_stop_cam">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>10</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spa_1_cam">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="bottomBox_cam">
            <item>
             <widget class="QComboBox" name="viewCbo_roi_cam">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>40</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>0.5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0.75</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.25</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.5</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="spa_2_cam">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
